fubectl.source

```diff
@@ -317,10 +316,10 @@ kupdate() {
 #### Kubermatic KKP specific
 # [kkp-cluster] Kubermatic KKP - extracts kubeconfig of user cluster and connects it in a new bash
 kkp-cluster () {
-	TMP_KUBECONFIG=$(mktemp)
-	local cluster="$(kubectl get cluster | _inline_fzf | awk '{print $1}')"
-	kubectl get secret admin-kubeconfig -n cluster-$cluster -o go-template='{{ index .data "kubeconfig" }}' | base64 -d > $TMP_KUBECONFIG
-	KUBECONFIG=$TMP_KUBECONFIG $SHELL
+  TMP_KUBECONFIG=$(mktemp)
+  local cluster="$(kubectl get cluster | _inline_fzf | awk '{print $1}')"
+  kubectl get secret admin-kubeconfig -n cluster-$cluster -o go-template='{{ index .data "kubeconfig" | base64decode }}' > $TMP_KUBECONFIG
+  KUBECONFIG=$TMP_KUBECONFIG $SHELL
```

Comment:
Please keep the original indentation

Reply:
This place had tab instead of spaces, so I've fixed it to have 4 spaces.
But the issue is that this file has inconsistend indendation - somewhere it's 2 spaces, somewhere it's 4 spaces.